spec/support/travel_advice_publisher_assertion_helpers.rb

```diff
@@ -0,0 +1,44 @@
+module TravelAdvicePublisherAssertionHelpers
+  def expect_create_new_edition
+    expect(page).to have_button("Create new edition")
+  end
+
+  def expect_new_edition(summary)
+    reload_page_while_failing do
+      within(".govuk-govspeak") do
+        expect(page).to have_content(summary)
+      end
+    end
+  end
+
+  def expect_published_edition(summary)
+    reload_page_while_failing do
+      within(".govuk-govspeak") do
+        expect(page).to have_content(summary)
+      end
+    end
+  end
+
+  def expect_new_part(part_body)
+    within(".govuk-govspeak") do
+      expect(page).to have_content(part_body)
+    end
+  end
+
+  def expect_file_attached
+    reload_page_while_failing do
+      expect(page).to have_link("Download map (PDF)")
+      expect(page).to have_xpath("//img[contains(@src,'world_map.png')]")
```

Comment:
It looks like this just tells us that there are links in the html but not that the files are actually there?